pkg/lib/pb/model/protos/models.proto

```diff
@@ -1157,21 +1157,7 @@ message ManifestInfo {
     string language = 12;
 }
 
-message Membership {
-    enum Tier {
-        TierNewUser = 0;
-        // "free" tier
-        TierExplorer = 1;
-        // this tier can be used just for testing in debug mode
-        // it will still create an active subscription, but with NO features
-        TierBuilder1WeekTEST = 2;
-        // this tier can be used just for testing in debug mode
-        // it will still create an active subscription, but with NO features
-        TierCoCreator1WeekTEST = 3;
-        TierBuilder = 4;
-        TierCoCreator = 5;
-    }
-    
+message Membership {    
     enum Status {
         StatusUnknown = 0;
         // please wait a bit more
```

Comment:
I don’t understand what it means "please wait a bit more"? wait on the client side for changing the status from StatusUnknown to another? there will be other statuses, I suppose?

Reply:
How subscription purchase works (in Stripe case):

* User requests a new subscription.
* User is redirected to Stripe.
* User makes the payment.
* Stripe immediately returns "success" and redirects you via a deeplink <-- at this point, the user may still not have the subscription.
* (Uncertain when) Within a certain period of time, Stripe sends us a webhook <-- at this point, the subscription will be created and will enter the **pending** state.
* (Uncertain when) We process it and load various settings: names, restrictions, etc.
Once everything is completed, the subscription transitions to the active state.

added more comments to the .proto file